ruby/decorator.rb

```diff
@@ -0,0 +1,33 @@
+require './nameable'
+require './person'
```

Comment:
- [ ] Change all the `require` to `require_relative`. It is a common practice to use relative when doing it for other classes which are present on your project

Reply:
hi @GzimAsani I have made the changes, although I think those are optional changes